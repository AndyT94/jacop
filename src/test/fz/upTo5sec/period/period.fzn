var 1..8: X_INTRODUCED_3;
var 1..8: X_INTRODUCED_4;
var 1..8: X_INTRODUCED_5;
array [1..8] of var int: x:: output_array([1..8]) = [X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_3,X_INTRODUCED_4];
array [1..3] of var int: the_period:: output_array([1..3]) = [X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5];
solve  satisfy;
