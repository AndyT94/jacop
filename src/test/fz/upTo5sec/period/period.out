x = array1d(1..8, [1, 1, 1, 1, 1, 1, 1, 1]);
the_period = array1d(1..3, [1, 1, 1]);
----------